src/util/invariant.h

```diff
@@ -12,7 +12,6 @@ Author: Martin Brain, martin.brain@diffblue.com
 #include <cstdlib>
 #include <stdexcept>
 #include <string>
-#include <tuple>
```

Comment:
Procedural nit-pick: do this in a separate commit. Possibly even a separate pull request.